File: back/src/main/java/com/myweather/api/services/models/CustomResponse.java
package com.myweather.api.services.models;

/**
 * Created by javierfz on 7/13/17.
 */
public class CustomResponse {

   private Boolean status;
   private String message;

   public Boolean getStatus() {
      return status;
   }

   public String getMessage() {
      return message;
   }

   public void setStatus(Boolean status) {
      this.status = status;
   }

   public void setMessage(String message) {
      this.message = message;
   }
}

File: front/src/app/shared/models/http/CustomResponse.ts
export class CustomResponse {

  private _status: string;
  private _message: string;
  private _data: any;

  set status(value: string) {
    this._status = value;
  }

  set message(value: string) {
    this._message = value;
  }

  set data(value: any) {
    this._data = value;
  }

  get status(): string {
    return this._status;
  }

  get message(): string {
    return this._message;
  }

  get data(): any {
    return this._data;
  }
}
